styles/prism-launcher/catppuccin.user.css

```diff
@@ -107,10 +107,18 @@
     --warn-dark: @yellow;
     --warn-light: @yellow;
 
+    .button:hover {
```

Comment:
The button should have a color of `crust` in general, not just on hover.

Reply:
Done.